tox.ini

```
@@ -26,6 +27,7 @@ deps =
 commands =
     python -m black --check src/
     python -m black --check tests/
+    python -m mypy --ignore-missing-imports src/nashpy/**
```

```suggestion
    python -m mypy --ignore-missing-imports src/nashpy
```

Apologies, this was my mistake in what I wrote before.